### ISSUE TITLE
Diff of event updates

### ISSUE THREAD

**Issue by nickvergessen** [nickv-nextcloud/event_update_notification#52]
With https://github.com/nextcloud/server/pull/35743#issuecomment-1370556968 it might be possible to improve the update notification to outline what changed (name, time, ...?)

**Comment by phil-lipp** [comment]
Yes, that would be great! The notifications right now are pretty bare bones and sometimes plain useless as I cant see what another person changed. Having 5 notifications of "Person A changed event X" and trying to figure out what each individual changed is not feasible most of the time.

**Comment by nickvergessen** [comment]
> Having 5 notifications of "Person A changed event X" and trying to figure out what each individual changed is not feasible most of the time.

Well depends of course. In my home usecase the mostlikely thing to change is the time and the starting time is also visible in the notifications, so for me the most important info is there.

But yeah I will look into adding at least the list of changed properties

**Comment by nickvergessen** [comment]
Hmm actually we don't receive the "before" data in the `CalendarObjectUpdatedEvent` event so we can also not diff it :(

**Comment by phil-lipp** [comment]
> Hmm actually we don't receive the "before" data in the `CalendarObjectUpdatedEvent` event so we can also not diff it :(

Well then can't that change? Is it not possible for the server to provide that info?

**Comment by nickvergessen** [comment]
It would be possible I guess, but required some work on multiple levels and I'm not familiar with that code base

**Comment by meonkeys** [comment]
I'm seeing a diff in "Invitation updated: TITLE" email notifications... maybe there's something there we could re-use?

<img width="581" height="295" alt="Image" src="https://github.com/user-attachments/assets/d3d92ae2-77a8-4f2a-b791-bb827a0a9086" />
